tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
+minversion = 1.6
+skipsdist = True
+
+[testenv]
+basepython = python3
+usedevelopr = False
+
+[testenv:pep8]
+whitelist_externals = flake8
```

Comment:
There should be `deps = flake8` so when creating `pep8` venv, `tox` will install the `flake8` tool. Then it is no longer an external command, so you can remove `whitelist_externals = flake8`.

Reply:
My mistake. Change done.